### PR TITLE
add slashes

### PR DESCRIPTION
**Pull request by KrishMunot**
### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by jayvdb** [comment]
Literal values should be quoted as literals.  See RST documentation.

File: Users/coafile.rst [hunk 1]
```diff
@@ -157,9 +157,9 @@ Similarly, you can also set a value to multiple keys:
 ``key_1, key_2 = value`` is equivalent to ``key_1 = value`` and
 ``key_2 = value`` in separate lines.
 
-As the backslash is the escape character it is recommended to use
-forward slashes as path separator even on Windows (to keep relative
-paths platform independent), use double-backslashes if you really mean a
+As the backslash (``\``) is the escape character it is recommended to use
+forward slashes (``/``) as path separator even on Windows (to keep relative
+paths platform independent), use double-backslashes (``\\``) if you really mean a
```
Comment (khanchi97) [inline Users/coafile.rst]: line length should not be greater than 79 characters

**Comment by palash25** [comment]
ping @KrishMunot can you please make the suggested changes

**Comment by jayvdb** [comment]
Abandoned PR.

**Pull request closed without merging**